pkg/kubectl/doc.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package kubectl is a set of libraries that are used by the kubectl command line tool.
-// They are separated out into a library to support unit testing.  Most functionality should
-// be included in this package, and the main kubectl should really just be an entry point.
+// Package kubectl provides the functions used by the kubectl command line tool
+// under /cmd/. The functions are kept in this package to better support unit
```

Comment:
Of course, I realize that. But I don't think it's a common pattern.

As an example:
```
$GOPATH/src/k8s.io/kubernetes$ cd pkg/kubectl
$GOPATH/src/k8s.io/kubernetes/pkg/kubectl$ git show HEAD:cmd/kubectl/
tree HEAD:cmd/kubectl

BUILD
OWNERS
app/
kubectl.go
$GOPATH/src/k8s.io/kubernetes/pkg/kubectl$ git show HEAD:/cmd/
fatal: Path '/cmd/' does not exist in 'HEAD'
```